ethereum/referencetests/src/reference-test/java/org/hyperledger/besu/ethereum/vm/GeneralStateReferenceTestTools.java

```diff
@@ -252,7 +252,7 @@ private static void logWorldState(final ReferenceTestWorldState worldState) {
               if (!storageEntries.isEmpty()) {
                 accountJson.set("storage", storageJson);
               }
-              worldStateJson.set(account.getAddress().orElse(Address.wrap(Bytes.EMPTY)).toHexString(), accountJson);
+              worldStateJson.set(account.getAddress().orElse(Address.ZERO).toHexString(), accountJson);
```

Comment:
is it better to do that 
```java
worldStateJson.set(account.getAddress().orElse(Bytes.EMPTY).toHexString()
```

Reply:
Done [`e50e2fb` (#8167)](https://github.com/hyperledger/besu/pull/8167/commits/e50e2fb4b29c6343175dc0908dd6847ca2366ea9)